.github/workflows/block-performance.yml

```diff
@@ -0,0 +1,116 @@
+name: Jetpack block performance
+
+on:
+  pull_request:
+    paths-ignore:
+    - '**.md'
+  push:
+    branches: [master]
+    paths-ignore:
+    - '**.md'
+  schedule:
+    - cron:  '0 */12 * * *'
+
+jobs:
+  block-performance:
+    name: "Performance tests"
+    runs-on: ubuntu-latest
+    timeout-minutes: 25  # 2021-04-16: Successful runs seem to take 11-17 minutes
+    env:
+      WP_USERNAME: wordpress
+      WP_PASSWORD: wordpress
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Setup tools
+      uses: ./.github/actions/tool-setup
+      with:
+        node: 14
+
+    - uses: actions/checkout@v2
+      with:
+        repository: 'WordPress/gutenberg'
+        path: 'gutenberg'
+
+    - name: Build Gutenberg
+      working-directory: gutenberg
+      run: |
+        npm ci
+        npm run build
+
+    - name: Build Production Jetpack
+      run: |
+        pnpm install
+        pnpx jetpack build plugins/jetpack -v --production
+
+    - name: Environment set-up
+      working-directory: tools/e2e-commons
+      env:
+        CONFIG_KEY: ${{ secrets.E2E_CONFIG_KEY }}
+      run: |
+        pnpm install
+        pnpm config:decrypt
+        echo "    gutenberg/packages/e2e-tests/plugins/disable-animations.php: /var/www/html/wp-content/plugins/disable-animations.php" >> ../docker/jetpack-docker-config-default.yml
```

Comment:
Gutenberg performance tests require the `disable-animations.php` plugin to be available on a site. Here we bluntly inject the required mapping into the default docker configuration. Probably not the best solution 😬

Reply:
A third option would be to copy it into `tools/docker/wordpress/wp-content/plugins/` after WordPress is installed.

> I wonder if instead of mapping each plugin individually we could map the whole `plugins` folder or its content.

If by that you mean that `projects/plugins/` should be mounted onto `/var/www/html/wp-content/plugins/`, that would probably result in the docker environment creating a bunch of garbage in the host's checkout of the monorepo, whenever any plugins are installed within the docker environment (e.g. Query Monitor, Core Control, WP Crontrol, and Gutenberg get installed by `jetpack docker install`).

Reply:
No, I meant `tools/e2e-commons/plugins`